test/Microsoft.DotNet.Cli.Sln.Internal.Tests/Microsoft.DotNet.Cli.Sln.Internal.Tests.cs

```diff
@@ -496,7 +496,15 @@ public void WhenGivenASolutionWithInvalidPropertySetItThrows()
             };
 
             action.ShouldThrow<InvalidSolutionFormatException>()
-                .WithMessage("Invalid format in line 7: Property set is missing '.'");
+                .WithMessage(FormatError(7, LocalizableStrings.InvalidPropertySetFormatString, "."));
+        }
+
+        private static string FormatError(int line, string format, params object[] args)
```

Comment:
NIT: I expected the format to come first here. Threw me off a bit that the line number came first.